src/nyc_trees/apps/mail/templates/mail/new_reservations_confirmed.txt

```diff
@@ -1,11 +1,14 @@
 {% extends "mail/base_email.txt" %}
 {% block content %}
-Your block edge reservations are confirmed!
+Thank you for signing up to map on your own during TreesCount! 2015.  The
+following block edges have been confirmed:
 
-The following block edges have been confirmed:
 {% for blockface in blockfaces %}
 * {{ blockface.id }}
 {% endfor %}
 
-Please see the attached map detailing your reservation.
+The block edges will be reserved for you to map until {{ expiration_date }}.
+
+You can view and change your other block edge reservations at
```

Comment:
since this reservation_url is part of a grammatical sentence, consider dropping the newline after "at".